client/src/routes/Session/IntroPortal.tsx

```diff
@@ -232,16 +232,17 @@ const IntroPortal: React.FC = () => {
             <PortalStatus>
               <StatusItem>
                 <StatusText themeColor={textColor}>
-                  {t('counterLabel.soon')}
+                  {t('counterLabel.starts')}
                 </StatusText>
 
                 <Spacer8 />
                 <Badge themeColor={textColor}>
                   <StatusText themeColor={textColor}>
-                    <Counter
-                      startTime={dayjs(session?.startTime.toDate())}
-                      starting={session?.started}
-                    />
+                    {session?.started ? (
+                      t('counterLabel.started')
+                    ) : (
+                      <Counter startTime={dayjs(session?.startTime.toDate())} />
```

Comment:
This is not new but maybe we could find a way to avoid `dayjs()` calls like this. It consumes memory to do this on every re render. It is not a problem here but could be in other places